README.md

```diff
@@ -1,14 +1,42 @@
-# Pollen
+# Pollen Operator
 
-More information: https://charmhub.io/pollen
+A juju charm deploying and managing Pollen on a machine. Pollen is a scalable, high performance, free software (AGPL) web server, that provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP connections. You might think of this as ‘Entropy-as-a-Service’.
 
-Pollen is a scalable, high performance, free software (AGPL) web server, that
-provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP
-connections. You might think of this as 'Entropy-as-a-Service'. This charm
-deploys a Pollen server.
+Pollinate is a free software (GPLv3) script that retrieves entropy from one or more Pollen servers and seeds the local Pseudo Random Number Generator (PRNG). You might think of this as PRNG-seeding via Entropy-as-a-Service.
```

Comment:
Maybe add a link to Pollinate? Maybe https://manpages.ubuntu.com/manpages/trusty/man1/pollinate.1.html 

I don't know if this is the best link; please let me know if you know of a more appropriate one!